edge-util/src/Microsoft.Azure.Devices.Edge.Storage/InMemoryDbStoreProvider.cs

```diff
@@ -28,5 +45,19 @@ public void Dispose()
         {
             // No-op
         }
+
+        long GetTotalMemoryUsage()
+        {
+            long dbSizeInBytes = 0;
+            foreach (var inMemoryDbStore in this.partitionDbStoreDictionary)
```

Comment:
I think this needs to take a snapshot because the dictionary might be changed while it iterates and this might throw

Reply:
Iterating over a ConcurrentDictionary is thread-safe and won't throw exceptions as per: https://stackoverflow.com/questions/17776422/is-this-the-proper-way-to-iterate-over-concurrentdictionary-in-c-sharp